### PR TITLE
Use dependabot for GitHub actions version management

### LINKED ISSUE #27816: Use dependabot for simplifying GitHub actions version management
### Description

GitHub action versions seems rarely updated. For instance `ci-actions-incremental.yml` is still using `action/checkout@v2` even if `action/checkout@v3` has been released a few months ago.

Dependabot is used in the project for managing some of the Maven and Gradle dependencies. But it can also manage GitHub actions version (package ecosystem `github-actions`).

Should Dependabot be used for managing GitHub actions versions ?

### Implementation ideas

Add the following to https://github.com/quarkusio/quarkus/blob/main/.github/dependabot.yml :

```yaml
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: daily
      time: "21:00"
      timezone: Europe/Paris
    open-pull-requests-limit: 4
    labels:
      - area/devtools
    rebase-strategy: disabled
```

### PR DESCRIPTION
**Pull request by marcwrobel**
Resolves: #27816

### CODE REVIEW

**Comment by marcwrobel** [comment]
@geoand, I am not sure which labels to use. Any idea ?

File: .github/dependabot.yml [hunk 1]
```diff
@@ -225,3 +225,13 @@ updates:
...
     rebase-strategy: disabled
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "21:00"
+      timezone: Europe/Paris
+    open-pull-requests-limit: 4
+    labels:
+      - area/devtools
```
Comment (gsmet) [inline .github/dependabot.yml:236]:
```suggestion
      - area/infra
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -225,3 +225,13 @@ updates:
     labels:
       - area/dependencies
     rebase-strategy: disabled
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "21:00"
+      timezone: Europe/Paris
+    open-pull-requests-limit: 4
```
Comment (gsmet) [inline .github/dependabot.yml:234]:
```suggestion
    open-pull-requests-limit: 2
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -225,3 +225,13 @@ updates:
     labels:
       - area/dependencies
     rebase-strategy: disabled
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "21:00"
```
Comment (gsmet) [inline .github/dependabot.yml:232]:
```suggestion
      time: "00:30"
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks, I added a few suggestions.

**Review by gsmet — APPROVED** [review]
Let's merge and see how it goes. Thanks!

**Pull request merged**